packages/react-components/react-badge/jest.config.js

```diff
@@ -13,7 +13,12 @@ module.exports = {
     },
   },
   transform: {
-    '^.+\\.tsx?$': 'ts-jest',
+    '^.+\\.tsx?$': [
+      '@swc/jest',
+      {
+        swcrc: false,
```

Comment:
we now use swc for transpilation. swcrc specifies `exclude` which will explode when executing test suites. 

Overriding exclude config imperatively doesn't work, this is only available workaround https://github.com/swc-project/jest/issues/62